packages/byline-parser/index.js

```diff
@@ -34,7 +34,8 @@ export default function (blobSource, { openingPattern, closingPattern, matchPatt
         throw new Error('No capture group is defined');
       }
 
-      tmp.push(captureGroup.slice(1));
+      if (!noGroup) tmp.push(captureGroup.slice(1));
+      if (noGroup) results.push(captureGroup.slice(1));
```

Comment:
Are you aware of that, go supports two syntax definitions for import?

### Single import
```go
import "os";
import "github.com/foo/bar";
```

### Multi imports
```go
import (
  "os"
  "github.com/foo/bar"
)
```

When I started with this PR, the single import was handled with a RegExp, like we do for other languages. This will match every import regardless of the value. 
While writing this PR, we don't support custom import paths and therefore I was thinking it would be better to prevent the link injection in the first place to avoid confusions. 

With 8f7cad0ddec4ea0fa38360a1625e084b94fe366e I restricted it for multi imports already. To implement that same behaviour for single imports, we have to use the RegExp builder like we do for mutli imports. 

Sorry, I should have explained it better, earlier. Does it make sense now?